src/components/Header/AccountLabel.js

```diff
@@ -0,0 +1,13 @@
+import styles from './Header.module.scss';
+
+export default function AccountLabel({ userName }) {
+  return (
+    <div className={styles.account_label}>
+      <div className={styles.account_label__user_name_div}>
+        <div className={styles.account_label__first_letter}>{userName.charAt(0).toUpperCase()}</div>
```

Comment:
When you have something to compute, extract it into a function/method, so it can be reusable and have a better file structure.